.github/workflows/wheels.yml

```diff
@@ -212,12 +213,53 @@ jobs:
           echo "CIBW_BUILD_FRONTEND=$CIBW" >> "$GITHUB_ENV"
 
       - name: Build wheels
+        if: matrix.buildplat[1] != 'win' && matrix.buildplat[2] != 'ARM64' # Skip for win_arm64 for manual build
         uses: pypa/cibuildwheel@cf078b0954f3fd08b8445a7bf2c3fb83ab3bb971 # v3.0.0b4
         env:
           CIBW_BUILD: ${{ matrix.python[0] }}-${{ matrix.buildplat[1] }}*
           CIBW_ARCHS: ${{ matrix.buildplat[2] }}
           CIBW_ENABLE: cpython-freethreading cpython-prerelease
 
+      # can be remove after https://github.com/actions/partner-runner-images/issues/90
+      - name: Install LLVM for win_arm64
+        if: matrix.buildplat[1] == 'win' && matrix.buildplat[2] == 'ARM64'
+        uses: KyleMayes/install-llvm-action@v2
+        id: install_llvm
+        with:
+          version: 20.1.6
+          force-url: https://github.com/llvm/llvm-project/releases/download/llvmorg-20.1.6/LLVM-20.1.6-woa64.exe
+          env: false # Do not set CC/CXX by the action          
+
+      # pkgconf and FortranRuntime hacks may be removed after fixes in according libraries
+      - name: Build wheel for win_arm64 manually
+        if: matrix.buildplat[1] == 'win' && matrix.buildplat[2] == 'ARM64'
+        shell: cmd
+        run: |
+          CALL "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" arm64
+
+          pip install build
+          pip install -r requirements/build.txt -r requirements/openblas.txt
+          python -c "import scipy_openblas32; print(scipy_openblas32.get_pkg_config())" > %GITHUB_WORKSPACE%\scipy-openblas.pc
+          
+          set VCPKG_ROOT=C:\vcpkg
+          vcpkg install pkgconf
+          set PATH=C:\vcpkg\installed\arm64-windows\tools\pkgconf;%PATH%
+          copy C:\vcpkg\installed\arm64-windows\tools\pkgconf\pkgconf.exe C:\vcpkg\installed\arm64-windows\tools\pkgconf\pkg-config.exe
+          
+          copy "C:\Program Files\LLVM\lib\FortranRuntime.static.lib" "C:\Program Files\LLVM\lib\FortranRuntime.lib"
+
+          set "PKG_CONFIG_PATH=%GITHUB_WORKSPACE%"
+
+          set "LLVM_LIB_DIRS=C:\Program Files\LLVM\lib\clang\20\lib\windows;C:\Program Files\LLVM\lib"
+          set "LIB=%LLVM_LIB_DIRS%;%LIB%"
+          set "LIBPATH=%LLVM_LIB_DIRS%;%LIBPATH%"
+
+          set CC=clang-cl
+          set CXX=clang-cl
+          set FC=flang
+
+          python -m build --wheel --no-isolation -o wheelhouse .
```

Comment:
This is incorrect. `cibuildwheel` should do the wheel building, you cannot invoke it manually, that will not result in a portable wheel.

For disabling isolation, you can do the same as in the step `Disable build isolation for python free-threaded version`.

Reply:
Yes, needs to go through cibuildwheel. I think that all the environment variable may need to be put in `GITHUB_PATH`/`GITHUB_ENV`.

Reply:
Got success with cibuildwheel, will update the PR soon